three.c

```diff
@@ -0,0 +1,109 @@
+/* Drew French */
+
+/* Takes three user inputs
```

Comment:
Nice job! Personally would have handled floats differently (not truncated them), but awesome job nevertheless.